buildpipeline/Core-Setup-Linux-Arm-BT.json

```diff
@@ -242,6 +206,16 @@
     {
       "enabled": false,
```

Comment:
Sounds good.

In general, it creates noise in reviews like this.  PR for "remove something" is adding new stuff to the file.  Seems odd.  But I'm not sure how to make it better.  Maybe separating out the "noise" from the real "changes" into separate commits?